### DISCUSSION TITLE
Rename File/Folder

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#251] [Ideas]
Hello,

Is it possible to add a feature to rename file/folder by the end user?
Thanks.

**Comment by rejetto** [comment]
yes, it's already in my to-do list

**Comment by rejetto** [comment]
is it ok if it's bound to the "delete" permission?
that is: if you want to be able to rename, you need delete permission.

**Reply by GeorgeGeorgeGe** [reply]
Yes, it works this way.
Thanks a lot.

**Comment by rejetto** [comment]
available in [0.46.0 alpha7](https://github.com/rejetto/hfs/discussions/257)

**Reply by GeorgeGeorgeGe** [reply]
it works perfect. but is it possible that the page is automatically refreshed after the renaming action, the rename is done but it is not reflected on the user page until he manually refreshes that page.

Thanks

**Reply by rejetto** [reply]
that was a bug.
fixed in beta8 #257

**Reply by GeorgeGeorgeGe** [reply]
Thanks a lot.
